cmd/tsort/main.go

```diff
@@ -0,0 +1,38 @@
+package main
```

Comment:
just a reminder to remove this file from the final PR

Reply:
Done